.template/addons/semaphore/template.rb

```diff
@@ -1,6 +1,3 @@
 use_source_path __dir__
 
 directory '.semaphore'
-
-template 'Dockerfile.web.tt'
-template 'Dockerfile.worker.tt'
```

Comment:
Correct me if I'm wrong, I think we need to keep this as the `Dockerfile.web.tt` and the `Dockerfile.worker.tt` still there

Reply:
I moved it into Heroku addon, as we're not only using it on Semaphore but also on Github action

Reply:
so do we still need these files @hoangmirs 🤔

![Pasted_Image_9_23_20__10_43](https://user-images.githubusercontent.com/11751745/93964428-ba875680-fd89-11ea-8ad5-48cfa87ffa5d.png)

Reply:
Deleted in https://github.com/nimblehq/rails-templates/pull/224/commits/a174f32b3cef4ec126c6cc05b4c518951b648910